compiler/rustc_error_messages/locales/en-US/borrowck.ftl

```diff
@@ -13,6 +13,6 @@ borrowck_could_not_normalize =
 
 borrowck_higher_ranked_subtype_error =
     higher-ranked subtype error
-  
+
 generic_does_not_live_long_enough =
```

Comment:
Could you do a drive-by fix here and change this slug so that it is `borrowck_generic_does_not_live_long_enough`? 

I should have caught this in the pull request that introduced it, we like to start the slugs with the crate name so that they don't overlap between files since it is all one big namespace.

Reply:
Done. Seems like I haven't completely understood yet how the translation from slug names to rust items works - the diagnostic used `borrowck::generic_does_not_live_long_enough` all along, so is it just that a leading `${crate_name}_` is chopped off during translation?

Reply:
Yeah, we get it from the file name and chop it off. But Fluent only knows the slug here, so when we go ask it to translate a message, if there were two `generic_does_not_live_long_enough` messages in different files, it would get confused.